### ISSUE TITLE
How can I replicate the results reported in the paper?

### ISSUE THREAD

**Issue by sidrakhandev** [jLooo/VADMamba#3]
Are pretrained weights available for any of the benchmark datasets? If not, are there specific training details (seed, epochs, LR schedule) needed for accurate reproduction?

Thank you for your help.

**Comment by jLooo** [comment]
We update the README.md and include the pretrained weight link.

**Comment by sidrakhandev** [comment]
Appreciated!
